FHIR-Converter/.vscodeignore

```diff
@@ -14,6 +14,7 @@ assets/**/*.gif
 azure-pipelines.yml
 client/.nyc_output/**
 client/coverage/**
+client/install-engine.sh
```

Comment:
Does `.vscodeignore` mean the files to ignore when installing the extension?

Reply:
These files will be ignored to reduce the size of the package when the extension is packaged.